include/secp256k1.h

```diff
@@ -852,8 +852,11 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_xonly_pubkey_tweak_add(
     const unsigned char *tweak32
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4) SECP256K1_ARG_NONNULL(5);
 
-/** Verifies that output_pubkey and has_square_y is the result of calling
- *  secp256k1_xonly_pubkey_tweak_add with internal_pubkey and tweak32.
+/** Tests that output_pubkey and has_square_y is the result of calling
+ *  secp256k1_xonly_pubkey_tweak_add with internal_pubkey and tweak32. Note
+ *  that this alone does _not_ verify anything cryptographically. If the tweak
```

Comment:
I don't know that "anything cryptographically" is. This sentence is hard to understand.

"Note that this does not verify that tweak32 is constructed in a specific way, e.g., as the hash of internal_pubkey and some message."
"Note that this does not verify that output_pubkey is a commitment to some message."

Reply:
fixed